service/src/test/java/bio/terra/drshub/services/DrsApiFactoryTest.java

```diff
@@ -38,43 +45,44 @@ private DrsProvider createDrsProvider() {
 
   @Test
   void testMTlsConfigured() {
-    var mTlsConfig = MTlsConfig.create().setCertPath("fake.crt").setKeyPath("fake.key");
+    var mTlsConfig = MTlsConfig.create();
+    when(restTemplateFactory.makeMTlsRestTemplateWithPooling(mTlsConfig)).thenReturn(restTemplate);
 
-    drsApiFactory.getApiFromUriComponents(
-        URI_COMPONENTS, createDrsProvider().setMTlsConfig(mTlsConfig));
+    var drsApi =
+        drsApiFactory.getApiFromUriComponents(
+            URI_COMPONENTS, createDrsProvider().setMTlsConfig(mTlsConfig));
+    assertThat(drsApi.getApiClient(), is(apiClient));
 
-    verify(restTemplateFactory).makeMTlsRestTemplateWithPooling(any(MTlsConfig.class));
-    verify(restTemplateFactory, never()).makeRestTemplateWithPooling();
+    verify(drsApiClientFactory).createClient(restTemplate);
```

Comment:
Actually I think this and the one on line 66 below are redundant due to strict stubbing.